README.md

```diff
@@ -149,3 +187,35 @@ so BT will know where the block is, and it can use the offset to seek in the dis
     }
 }
 ```
+
+
+## Limitation
+
+
+- Making a torrent cost lots of time due to sha-1 hash need to be done on every single piece of data.
```

Comment:
I am working on partclone to dump filesystem and create torrent in the same time